### PR TITLE
Edited instructions.

### PR DESCRIPTION
**Pull request by JacOng17**
Editing instructions on how to setup/get started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,71 +32,130 @@ Scrum Master | [@merridy](https://github.com/merridy)
...
 ## Comms
+
 Slack: LabPlus-team #feijoa
 
 ## Setup
 
+### Prerequisites
+
+- Postgres
+- Java
```
Comment (mermop) [inline README.md:43]: Java?? What for? Is that for elasticsearch?

File: README.md [hunk 2]
```diff
@@ -30,71 +32,130 @@ Scrum Master | [@merridy](https://github.com/merridy)
...
 On Mac you may want to use Homebrew:
-```
+
+```bash
 brew install elasticsearch
 brew services start elasticsearch
 ```
 
+**To check whether Elastic Search is running (or not)**
+`sudo service elasticsearch status`
```
Comment (mermop) [inline README.md]: Is `sudo` necessary here?

File: README.md [hunk 3]
```diff
@@ -30,71 +32,130 @@ Scrum Master | [@merridy](https://github.com/merridy)
...
+bundle exec rspec
 ```
 
 Test coverage is reported to `coverage/index.html`
+
+---
+
+## Script to create users/organisations, etc...in **rails console:**
+
+`rails runner bin/runner`
```
Comment (mermop) [inline README.md]: This is straight in the terminal isn't it?

File: README.md [hunk 4]
```diff
@@ -30,71 +32,130 @@ Scrum Master | [@merridy](https://github.com/merridy)
...
+Eg.
+`user.add_role(organisation, OrganisationMember::ADMIN_ROLE)`
+
+---
+
+## Adding Birth Records & Immunisation Records
+
+Login using email: `123@123.nz`, password: `P@ssw0rd`
+
+Add the Birth Records & Immunisation Records to the user (as many as you want from the 10 generated) by *clicking on* **Add a document**.
```
Comment (mermop) [inline README.md:149]: Isn't there a script for this?

File: bin/runner [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+user = FactoryBot.create :user, email: '123@123.nz', password: 'P@ssw0rd'
```
Comment (mermop) [inline bin/runner]: You could possibly give this a more descriptive name than `runner` - maybe `user_seed`?

File: README.md (hunk 1 above)
Comment (JacOng17) [reply README.md:43]: Yup, https://tecadmin.net/setup-elasticsearch-on-ubuntu/
I needed to install it for my "fresh" machine.

File: README.md (hunk 2 above)
Comment (JacOng17) [reply README.md]: Good catch, not needed :stuck_out_tongue:

File: README.md (hunk 3 above)
Comment (JacOng17) [reply README.md]: Yup :+1:

File: README.md (hunk 4 above)
Comment (JacOng17) [reply README.md:149]: The script is the `bin/runner`

File: bin/runner (hunk 5 above)
Comment (JacOng17) [reply bin/runner]: Yup, will do :+1:

**Review by mermop — APPROVED** [review]

**Pull request merged**